canvas_modules/common-canvas/src/common-properties/form/EditorForm.js

```diff
@@ -409,6 +409,7 @@ function _makeControl(parameterMetadata, paramName, group, structureDefinition,
 	let moveableRows = parameter.moveableRows;
 	let rowSelection;
 	let addRemoveRows;
+	let hideEditButton;
```

Comment:
Are these used anymore?

Reply:
logic in Editorform for hideEditButton during the time to compose EditorForm is removed